Aciroh/iMarketlot
# Configuration Form for Windows Forms App

#### Description
 
Add a form that can configure the WinForms app. Also make the Config Form run only if and when the Config file does not exist in the Resource Folder.

 #### [Issue](https://github.com/Aciroh/iMarketlot/issues/3)
 

 ## Steps to Test or Reproduce
 
The first time you run the app, the form config should automatically start. Play with it then click on the Done button to get to the next form.

 ## Risks
 
 Absolutely none. Except probably Cross Threading or too many Threads which might slow the machine down?
